kahuna/public/js/components/gr-add-preset-label/gr-add-preset-label.css

```diff
@@ -0,0 +1,31 @@
+gr-add-preset-label{
+    display: flex;
+    height: 25px;
+    padding: 5px;
+}
+
+gr-add-preset-label:hover {
+    color: white;
+}
+
+gr-add-preset-label .small {
+    border: none;
+    padding: 0;
+}
+
+.gr-add-preset-label__form {
+    display: flex;
+}
+
+.gr-add-preset-label__form__input {
+    width: 150px;
+}
+
+.gr-add-preset-label__form__buttons {
```

Comment:
BEM is only really effective with one layer deep, otherwise it almost destroys it's own usefulness.

You could have something like `gr-add-preset-label__form-input` or to be more descriptive `gr-add-preset-label__label-input`.